src/ontology/mondo-edit.obo

```diff
@@ -433256,6 +433265,15 @@ intersection_of: MONDO:0000508 ! syndromic intellectual disability
 intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/21637 ! SATB2
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100150
+name: RYR1-related myopathy
+comment: A disorder of the musculoskeletal system caused by pathogenic variants in the RYR1 gene, which encodes the ryanodine receptor type 1 protein. These variants are associated with a variety of overlapping features characterized by symmetric proximal muscle weakness, often with pronounced facial weakness with or without dysmorphism and ophthalmoparesis/ophthalmoplegia with ptosis, bulbar weakness, significant respiratory involvement, severe neonatal hypotonia, scoliosis, orthopedic deformities including arthrogryposis, hip dislocation, club feet, and King Denborough syndrome (pectus carinatum or excavatum, short stature, joint contractures, facial and skeletal deformities), malignant hyperthermia susceptibility, anesthesia-induced rhabdomyolysis, fatigue, exercise-induced hyperthermia/exertional heat stroke, and exertional myalgia. Histologic findings on skeletal muscle biopsy reveal a wide range of structural abnormalities and can include central core disease, multiminicore disease, cone-rod myopathy, centronuclear myopathy, and congenital fiber-type disproportion. {xref="https://clinicalgenome.org/affiliation/40031/", xref="PMID:30406384"}
+is_a: MONDO:0019952 {source="https://clinicalgenome.org/affiliation/40031/"} ! congenital myopathy
+intersection_of: MONDO:0019952 {source="https://clinicalgenome.org/affiliation/40031/"} ! congenital myopathy
```

Comment:
what do you see in ELK when you add this? What is the inferred relation to:

```
id: MONDO:0009712
name: congenital multicore myopathy with external ophthalmoplegia
def: "An autosomal recessive condition caused by mutation(s) in the RYR1 gene, encoding ryanodine receptor 1. It may be characterized clinically by neonatal hypotonia, delayed motor development, and generalized muscle weakness, and amyotrophy. Pathologically, the absence of mitochondria and focal disorganization of the sarcomere appear as \"minicores\" on ATPase staining as a result of focal defects in oxidative activity." [NCIT:C150608]
subset: ordo_clinical_subtype {source="Orphanet:98905"}
synonym: "minicore myopathy" RELATED [OMIM:255320]
synonym: "minicore myopathy with external ophthalmoplegia" RELATED [OMIM:255320]
synonym: "minicore myopathy, antenatal onset, with arthrogryposis" EXACT [OMIM:255320]
synonym: "multicore myopathy" RELATED [OMIM:255320]
synonym: "multicore myopathy with external ophthalmoplegia" RELATED [GARD:0010316]
synonym: "multiminicore disease with external ophthalmoplegia" RELATED [OMIM:255320]
synonym: "multiminicore myopathy multicore myopathy with external ophthalmoplegia" RELATED [OMIM:255320]
xref: GARD:0010316 {source="MONDO:equivalentTo", source="Orphanet-shared", source="OMIM-shared"}
xref: ICD10:G71.2 {source="ORDO:98905/attributed", source="ORDO:98905/ntbt", source="Orphanet:98905"}
xref: NCIT:C150608 {source="MONDO:equivalentTo"}
xref: OMIM:255320 {source="MONDO:equivalentTo", source="ORDO:98905/e", source="Orphanet:98905"}
xref: Orphanet:598 {source="MONDO:subClassOf", source="OMIM:255320"}
xref: Orphanet:98905 {source="MONDO:equivalentTo", source="OMIM:255320"}
xref: UMLS:C1850674 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo", source="OMIM:255320", source="Orphanet:98905"}
is_a: MONDO:0018948 {source="Orphanet:98905"} ! multiminicore myopathy
relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/10483 {source="mim2gene_medgen"} ! RYR1
```

Reply:
ah I see, it looks like you asserted this as a child term, as requested in the ticket

Reply:
yes, I asserted it.

It  would be inferred to  be a child of 'RYR1-related myopathy' though if it wasn't asserted (I just tested that out.)